dll/win32/browseui/shellfind/CFindFolder.cpp

```diff
@@ -671,11 +671,17 @@ STDMETHODIMP CFindFolder::GetDetailsOf(PCUITEMID_CHILD pidl, UINT iColumn, SHELL
     if (!pidl)
         return SHSetStrRet(&pDetails->str, _AtlBaseModule.GetResourceInstance(), g_ColumnDefs[iColumn].iResource);
 
-    if (iColumn == 1)
+    if (iColumn == IDS_COL_LOCATION_COLNUM)
     {
         return SHSetStrRet(&pDetails->str, _ILGetPath(pidl));
     }
 
+    if (iColumn == IDS_COL_RELEVANCE_COLNUM)
+    {
+        /* TODO: Fill once the relevance is calculated */
+        return SHSetStrRet(&pDetails->str, "");
+    }
+
     return GetDisplayNameOf(pidl, SHGDN_NORMAL, &pDetails->str);
```

Comment:
Shouldn't this even be some kind of `if (iColumn == the_name_column_index) { return GetDisplayNameOf(...); }` , in order to ensure always that the file name will only be shown in the "Name" column only?

Reply:
I see some other `GetDetailsOf` implementations return a failure status for unknown column ids. That seems appropriate, but the current implementation doesn't seem like it would really hurt, either. This is pre-existing behavior so I'd be okay with just leaving it as is for this PR.

Reply:
Oh good catch, that will prevent name being added to any new columns which are introduced.

Although you are right, i would prefer to do it as a seperate commit since this PR targets a specific issue.